hera_cal/vis_clean.py

```diff
@@ -1106,7 +1106,7 @@ def fourier_filter(self, filter_centers, filter_half_widths, mode='clean',
                     xp[1] = xp[1][spw_slice]
                     if not isinstance(zeropad, (list, tuple)) or not len(zeropad) == 2:
                         raise ValueError("zeropad must be a 2-tuple or 2-list of integers")
-                    if not (isinstance(zeropad[0], (int, np.int)) and isinstance(zeropad[0], (int, np.int))):
+                    if not (isinstance(zeropad[0], (int, int)) and isinstance(zeropad[0], (int, int))):
```

Comment:
Here as well. Looks like the second item should index the first element of `zeropad`, too. You could also do a comprehension: `if not all (isinstance(x, int) for x in zeropad)`.

Reply:
Good catch. Fixed.